tests/kernel/workq/work_queue/src/main.c

```diff
@@ -351,48 +352,42 @@ static int test_delayed(void)
 	return check_results(NUM_TEST_ITEMS);
 }
 
-
-void main(void)
+void testing_workq(void)
 {
-	int status = TC_FAIL;
+	int old_prio = k_thread_priority_get(k_current_get());
```

Comment:
why do we need this?

Reply:
I was trying to preserve priority because I have seen similar practice in some networking case. So I followed them. I will remove this.